apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/mapOrders.ts

```diff
@@ -0,0 +1,29 @@
+import {SwapOpenOrder} from '@yoroi/types/lib/swap/order'
+import React from 'react'
+
+import {getMockOrders} from './mocks'
```

Comment:
is it temporary ?

Reply:
@MichalSzorad and i are going to implement the api for our current task